modules/react/layout/stories/Flex.stories.mdx

```diff
@@ -23,6 +24,8 @@ yarn add @workday/canvas-kit-react
 
 ## Usage
 
```

Comment:
Can we add more context here on how to use this, maybe even a snippet of an example?